editor/components/editor-history/redo.js

```diff
@@ -6,11 +6,17 @@ import { IconButton } from '@wordpress/components';
 import { withSelect, withDispatch } from '@wordpress/data';
 import { compose } from '@wordpress/element';
 
+/**
+ * Internal dependencies
+ */
+import { primaryShortcut } from 'utils/keycodes';
+
 function EditorHistoryRedo( { hasRedo, redo } ) {
 	return (
 		<IconButton
 			icon="redo"
 			label={ __( 'Redo' ) }
+			shortcut={ primaryShortcut( 'Y' ) }
```

Comment:
`Shift+Command+Y` is the convention for "Redo" on Mac, so I wonder if we need yet another conditional helper here 😧

<img width="244" alt="screenshot 2018-05-05 15 07 14" src="https://user-images.githubusercontent.com/90871/39664104-2bf62e54-5076-11e8-9760-d5bdda824989.png">

`Command+Y` in Safari will actually change the current page to the History page, even when a textarea is focused:
<img width="496" alt="screenshot 2018-05-05 15 09 46" src="https://user-images.githubusercontent.com/90871/39664117-68bd6ca8-5076-11e8-8540-7adf6927bbd4.png">

So this probably needs to be `Shift+Command+Y`, at least on MacOS.

Reply:
Interesting, didn't know about the Safari command clash. Interestingly this depends on the application:

<img width="462" alt="screen shot 2018-05-05 at 18 51 50" src="https://user-images.githubusercontent.com/4710635/39665483-78d274ca-5095-11e8-82d1-5e4a296527d0.png">

Does SHIFT + PRIMARY + Z also make sense on Windows and Linux?

Reply:
Actually, I spoke too soon. Edge also supports `Primary+Shift+Z` for redo in Gutenberg. `Primary+Y` is the default but it seems Edge (and all other browsers) also work with `Primary+Shift+Z`. I'm just testing Linux now but it works for Mac+Win...

Reply:
Linux works with `Primary+Shift+Z` too, so let's just use that one everywhere.

Reply:
Okay. Wondering if we should still keep `Primary+Y` then as we are blocking the browser shortcuts on Mac.